scripts/bo_benchmarks/demonstrations/branin_campaign_demonstration.py

```diff
@@ -0,0 +1,348 @@
+#!/usr/bin/env python3
+"""
+Branin optimization campaign with 10 trials and iteration-by-iteration evaluation metrics calculation.
+This script demonstrates proper use of ax-platform with GP evaluation metrics calculated after each trial.
+"""
+
+import numpy as np
+import pandas as pd
+from ax.service.ax_client import AxClient, ObjectiveProperties
+from ax.adapter.cross_validation import cross_validate
+import matplotlib.pyplot as plt
+import torch
+from gpcheck.models import GPModel, GPConfig
+from gpcheck.metrics import loo_pseudo_likelihood, importance_concentration
+from sklearn.metrics import r2_score
+from scipy.stats import kendalltau
+
+obj1_name = "branin"
+
+
+def branin(x1, x2):
+    y = float(
+        (x2 - 5.1 / (4 * np.pi**2) * x1**2 + 5.0 / np.pi * x1 - 6.0) ** 2
+        + 10 * (1 - 1.0 / (8 * np.pi)) * np.cos(x1)
+        + 10
+    )
+    return y
+
+
+def interval_score(y_true, lower, upper, alpha=0.05):
+    """
+    Calculate interval score for uncertainty quantification evaluation.
+    
+    For 95% confidence intervals (alpha=0.05), the formula is:
+    f_interval = (upper - lower) + (2/alpha) * max(0, lower - y) + (2/alpha) * max(0, y - upper)
+    
+    Args:
+        y_true: True values
+        lower: Lower confidence bounds
+        upper: Upper confidence bounds 
+        alpha: Confidence level (0.05 for 95% CI)
+    
+    Returns:
+        Array of interval scores (lower is better)
+    """
+    width = upper - lower
+    below_penalty = np.maximum(0, lower - y_true)
+    above_penalty = np.maximum(0, y_true - upper)
+    return width + (2 / alpha) * below_penalty + (2 / alpha) * above_penalty
+
+
+def normal_95ci(mean, std):
+    """Convert mean and standard deviation to 95% confidence intervals."""
+    lower = mean - 1.96 * std
+    upper = mean + 1.96 * std
+    return lower, upper
+
+
+def calculate_ax_cross_validation_metrics(ax_client):
+    """Calculate metrics using Ax's built-in cross validation."""
+    try:
+        # Skip if using Sobol (first 5 trials) - no model available yet
+        if ax_client.generation_strategy.current_step_index == 0:
+            return None, None
+            
+        # Get the adapter from the generation strategy - this is the correct approach
+        adapter = ax_client.generation_strategy.adapter
+        if adapter is None:
+            return None, None
+        
+        # Run cross validation using Ax's built-in function with the adapter
+        cv_results = cross_validate(model=adapter, folds=-1)  # -1 for leave-one-out
```

Comment:
Install ax-platform<1, and if that is causing issues, then maybe you need to drop to `0.4.3`.

```python
from ax.modelbridge.cross_validation import cross_validate
...
cv = cross_validate(model)
fig = interact_cross_validation_plotly(cv)
y_act = fig["data"][1].x
y_pred = fig["data"][1].y
```